docs/guide/instrumentation.md

```diff
@@ -15,14 +15,19 @@ To enable ActiveSupport notifications, use the `instrumentation_enabled` option:
 # config/application.rb
 # Enable ActiveSupport notifications for all ViewComponents
 config.view_component.instrumentation_enabled = true
+config.view_component.use_deprecated_instrumentation_name = false
 ```
 
-Then subscribe to the `!render.view_component` event:
+Setting `use_deprecated_instrumentation_name` configures the event name. If `false` the name is `"render.view_component"`. If `true` (default) the deprecated `"!render.view_component"` will be used.
+
+Subscribe to the event:
 
 ```ruby
-ActiveSupport::Notifications.subscribe("!render.view_component") do |*args|
+ActiveSupport::Notifications.subscribe("render.view_component") do |*args| # or !render.view_component
   event = ActiveSupport::Notifications::Event.new(*args)
-  event.name    # => "!render.view_component"
+  event.name    # => "render.view_component"
   event.payload # => { name: "MyComponent", identifier: "/Users/mona/project/app/components/my_component.rb" }
 end
 ```
+
+When using `render.view_component` with `config.server_timing = true` (default in development) in Rails 7, the Chrome developer tools display the sum total timing information in Network > Timing under the key `render.view_component`.
```

Comment:
Not important but: specifying Chrome here is perhaps a little "chrome-centric".  The feature works nearly the same in FF, Safari, Edge and Opera.  The screenshot in the PR is from FF 113 IIRC.

Reply:
Is there anyway you can add Firefox and safari instructions

Reply:
They are nearly identical aside from coloration, so I don't think it's necessary.  

1. Network tab (refresh if its empty)
2. Click the request you're interested in.
3. on the right-side OR the bottom-side (depends on size of the dev-tools area) you'll see several tabs for that request; choose "Timing" **or "Timings" (in FF)**
4. Look for server timings at the bottom beneath the client-side data.

Biggest caveats with Server-Timings in general:
* It's not provided for every request.  Normal MVC rails controllers in development mode will provide it (it's a middleware) - but assets won't.  In production you won't see it without enabling it in production.rb.
* It does not give you the level of detail (or cross-request metrics) you can get from a product like Skylight or New Relic.  I'd suggest using it for quick-checks mainly, e.g.: "huh active record is crazy on this one, maybe I should take a look at my queries."
* Does not exist prior to Rails 7.

![timings](https://github.com/ViewComponent/view_component/assets/1916144/6222780c-c834-48a3-a06b-c34fec99c7d6)
From top-left going clockwise: Chrome, Firefox, Safari.  You can see its pretty much the same.

Reply:
How about we just say "browser here"? Perhaps we should add this screenshot to the docs?